ledger-core/network/messagesender/router.go

```diff
@@ -6,11 +6,13 @@
 package messagesender
 
 import (
+	"github.com/ThreeDotsLabs/watermill/message"
 	"github.com/insolar/assured-ledger/ledger-core/appctl/affinity"
 	"github.com/insolar/assured-ledger/ledger-core/appctl/beat"
 )
 
 type MessageRouter interface {
 	CreateMessageSender(affinity.Helper, beat.History) Service
 	SubscribeForMessages(func(beat.Message) error) (stopFn func())
+	PublishMessage(topic string, msg *message.Message) error
```

Comment:
this operation belongs to Service, returned by CreateMessageSender

Reply:
it is not Service method. This is proxy to internal Publisher it has different interface

Reply:
I'll remove this method from this interface at all